packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -705,7 +705,7 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
     private readonly summarizerNode: IRootSummarizerNodeWithGC;
 
     private _orderSequentiallyCalls: number = 0;
-    private _flushMode = FlushMode.Automatic;
+    private _flushMode = FlushMode.TurnBased;
```

Comment:
Have we scout out with our partners how changing this default will impact them?

Reply:
Not really. I don't have any good ideas about how to gauge this, searching through the code isn't viable as it's a default so any assumption baked into the logic would only be rooted out for sure when the FF version is upgraded. There is a quick workaround (setting it back to `Automatic`) to unblock, so that's fortunate. I did do a code search and the runtime does not have FlushMode.Manual set explicitly so I guess we may have some impact.

Do you have any recommendations here? Contacts I could reach out to?

Reply:
Note that flipping back to Automatic works only for the time BatchManager is around. So we should give it at least a version between such changes.

I do not think anybody will be able to tell the answer to that question. We can ping wider forum (general channel), but I think we will learn more by doing. batchEnd events/markers that Navin raised is definitely is something we have to have an opinion on (RE how to address these - see my list). Given that it likely requires some changes in app schema, it means we are likely looking at very slow propagation of changes across time.